frontend/src/app/aethel/aethel.component.scss

```diff
@@ -0,0 +1,27 @@
+.table-container {
+    margin-top: 2em;
+}
+
+.aethel-input-container {
+    display: flex;
+    gap: 1em;
+}
+
+.no-results {
+    text-align: center;
+}
+
+.expand-button {
+    width: 40px;
+}
+
+.expanded-row .button-container {
+    position: relative;
+
+    button {
```

Comment:
Centering the button using flexbox did not work in this PrimeNG/expanded row construction, so I had to resort to an old-fashioned way of doing it.